google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/doc/google/devtools/clouderrorreporting/v1beta1/common.rb

```diff
@@ -15,6 +15,34 @@
 module Google
   module Devtools
     module Clouderrorreporting
+      ##
+      # The `Google::Devtools::Clouderrorreporting::V1beta1` module provides the following types:
+      #
+      # Class | Description
+      # ----- | -----------
+      # {Google::Devtools::Clouderrorreporting::V1beta1::DeleteEventsRequest} | Deletes all events in the project.
```

Comment:
Most of the request object descriptions are of the form "specifies abc". This one is an odd one out, and a bit confusing: the object itself doesn't delete events; it's a request object for an API call that deletes events. Can we standardize the language a bit?